include/mppic/trajectory_visualizer.hpp

```diff
@@ -23,12 +23,8 @@ class TrajectoryVisualizer
   void on_activate();
   void on_deactivate();
 
-  void reset();
-
-  void add(xt::xtensor<double, 2> trajectory);
-
-  void add(xt::xtensor<double, 3> trajectories, size_t batch_step, size_t time_step);
-
+  void add(const xt::xtensor<double, 1> & trajectory);
```

Comment:
Are these the right types? I did xt::xtensor<double, 2> and xt::xtensor<double, 3> since I tracked down the dimensions from the functions that were inputs to `add`. Perhaps I made a mistake though.